src/sagasBuilder.js

```diff
@@ -0,0 +1,5 @@
+import { buildSagas as VmActionsSagas } from './components/VmActions/sagas'
+
+export default (sagas) => [
```

Comment:
This is becoming one of the key parts of the overall architecture.
Please add comment describing it's purpose, try to be as much descriptive as possible.
Try to describe the general idea behind this composition and what is expected to be done for future additions.